minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -144,8 +144,8 @@ public void start(int timeoutSeconds) {
     /**
      * Prints the state of the Mesos master or agent
      *
-     * @param out TODO(mwl)
-     * @param agentContainerId TODO(mwl)
+     * @param out A PrintStream to push the output to
+     * @param agentContainerId The Id of a mesos agent container. If null then the state of the master will be printed
```

Comment:
Pro tip: Start the sentence with _Optional_. There's nothing better when you're hacking together something and you see first up that something is optional :)